pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -2841,15 +2792,6 @@ func TestPodTimestamp(t *testing.T) {
 			operands: []*framework.QueuedPodInfo{pInfo2, pInfo1},
 			expected: []*framework.QueuedPodInfo{pInfo1, pInfo2},
 		},
-		{
-			name: "update two pod to activeQ and sort them by the timestamp",
-			operations: []operation{
-				updatePodActiveQ,
-				updatePodActiveQ,
-			},
-			operands: []*framework.QueuedPodInfo{pInfo2, pInfo1},
-			expected: []*framework.QueuedPodInfo{pInfo1, pInfo2},
-		},
```

Comment:
why was this deleted?

Reply:
It's now duplicated with the "add two pod to activeQ and sort them by the timestamp" test above, because `heap.Add` and `heap.Update` were unified to `heap.AddAndUpdate`